parallel-install/pkg/deployment/deployment.go

```diff
@@ -134,6 +136,10 @@ func (i *Deployment) StartKymaUninstallation() error {
 	return nil
 }
 
```

Comment:
Please add a godoc comment like above (e.g. `//StartKymaDeployment implements the Installer.StartKymaDeployment contract.`)